ui/src/logs/components/LogsTable.tsx

```diff
@@ -116,7 +147,7 @@ class LogsTable extends PureComponent<Props, State> {
 
     switch (column) {
       case 'message':
-        return 900
+        return 1200
```

Comment:
consider making these constants or enums

Reply:
Agree they should be constants.